src/sentry/api/endpoints/organization_integration_repos.py

```diff
@@ -44,6 +45,11 @@ def get(
             context = {"repos": []}
             return self.respond(context)
 
+        installed_repos = Repository.objects.filter(integration_id=integration.id).exclude(
+            status=ObjectStatus.HIDDEN
+        )
+        external_names = {hidden_repo.name for hidden_repo in installed_repos}
```

Comment:
is it misleading to call these `hidden_repo`s since we're actually excluding repos that are hidden in this list?

Reply:
oh yeah my b